maestro-ios-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -182,10 +189,11 @@ class XCTestDriverClient(
     }
 
     fun setPermissions(permissions: Map<String, String>) {
-        executeJsonRequest("setPermissions", SetPermissionsRequest(permissions))
+        executeJsonRequestUNCHECKED("setPermissions", SetPermissionsRequest(permissions))
     }
 
-    private fun executeJsonRequest(pathSegment: String, body: Any): Response {
+
+    private fun executeJsonRequestUNCHECKED(pathSegment: String, body: Any): Response {
```

Comment:
This is actually unchecked because we are not closing the response body in these calls, which was actually case for every call to iOS XCUITest driver. We have added a new method `executeJsonRequest` to ensure we use that for any new call.

So, there is no such exception handled here. Let me know if that clears the naming?